frontend/src/Dashboard/HistoricData.js

```diff
@@ -41,10 +48,40 @@ export default function HistoricData() {
     fetchData();
   }, [id]);
 
+  useEffect(() => {
+    async function fetchData() {
+      if (station.length > 0) {
+        try {
+          const response = await axios.get(`/api/latest?stationId=${station}`);
+          response.data && setWeatherData(response.data);
+        } catch (err) {
+          console.error(err);
+        }
+      }
+    }
+    fetchData();
+  }, [station]);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await axios.get(`/api/hasdata`);
```

Comment:
A /api/hasdata-tól nem tudom igazán mit is várok. Nem lehetne ennek jobb nevet adni?

Reply:
Ha állomásokat kapok, akkor legyen /api/stations. Ha van még egy szűrés a queryben hogy legyen adata, akkor meg /api/stations/hasData vagy még jobb a /api/stations?hasData=true

Reply:
Ezzel még lógok, majd otthon